### ISSUE TITLE
Dice loss

### ISSUE THREAD

**Issue by tracob-github** [cwmok/Conditional_LapIRN#7]
Dear author, 
      Thank you for this excellent work. I have seen that you mentioned dice loss in the [Presentation](https://learn2reg.grand-challenge.org/Learn2Reg2020/), but why is it not reflected in this repo? Looking forward to your reply.

**Comment by cwmok** [comment]
Hi @tracob-github,

This repository is for the implementation of our paper in MICCAI: 

- Conditional Deformable Image Registration with Convolutional Neural Network
[Tony C. W. Mok](https://cwmok.github.io/), Albert C. S. Chung
MICCAI 2021. [eprint arXiv:2106.12673](https://arxiv.org/abs/2106.12673)

In the original paper, the training is fully unsupervised; therefore, the dice loss is not included in this repository.

**Comment by tracob-github** [comment]
Thank you for your reply. I found that dice loss was added to Task1 and Task3 in your presentation. Did you add dice loss in the final implementation? 
![121ea7e2af4be1477b07262356c544e](https://user-images.githubusercontent.com/59277274/197311246-f3fcfdf7-2c23-4f4b-bc8f-5c221bca4721.jpg)

![960336c8fa70ba1bc82ff011390c59e](https://user-images.githubusercontent.com/59277274/197311255-e435f13f-6cf3-4ebd-b937-1b48b8d8eb25.jpg)

Looking forward to your reply.

**Comment by cwmok** [comment]
Hi @tracob-github,

We did use dice loss during the learn2reg challenge in order to maximize the performance. For more details of our approach in the challenge, you may refer to our challenge paper at https://link.springer.com/chapter/10.1007/978-3-030-97281-3_23.

**Comment by tracob-github** [comment]
Thank you for your patient reply. And is it convenient for you to provide the source code with dice loss? I want to learn about weakly-supervised registration. Looking forward to your reply.

**Comment by cwmok** [comment]
Hi @tracob-github,

Example of the dice loss https://github.com/voxelmorph/voxelmorph/blob/793b90fbddfd93f31019ecbb8c036454c6ee0720/voxelmorph/torch/losses.py#L79

**Comment by tracob-github** [comment]
Thank you very much. It helped me a lot.
